### PR TITLE
rds-command-line-tools: install credential-file-path.template to etc directory

### PR DESCRIPTION
**Pull request by kyanny**
- Install `credential-file-path.template` file to etc directory
- Add explanation about this file to caveats section

### CODE REVIEW

File: Library/Formula/rds-command-line-tools.rb [hunk 1]
```diff
@@ -25,6 +26,9 @@ def caveats
         export AWS_SECRET_KEY="<Your AWS Secret Key>"
         export AWS_CREDENTIAL_FILE="<Path to the credentials file>"
 
+      You can find a template of credential file at `#{etc}/credential-file-path.template`.
+      Edit a copy of this file to add your AWS keys.
```
Comment (mikemcquaid) [inline Library/Formula/rds-command-line-tools.rb]: Don't think we need the caveats, thanks

Comment (kyanny) [inline Library/Formula/rds-command-line-tools.rb]: @mikemcquaid Deleted. Are you happy? 228dd46

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by kyanny** [comment]
Your welcome! :smile:

**Pull request closed without merging**